dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -28,14 +28,14 @@ RUN apt-get update &&       \
 # Fix for gcc/python/iputils-ping not found in arm docker
 RUN apt-get install -f -y python2.7 python2.7-dev
 RUN apt-get install -y   \
-        gcc-8            \
+        gcc-10           \
```

Comment:
Does it need to be a specific version of GCC? Can it just be `gcc` here?

Reply:
Actually, I realized that Python 2.7 is no logner available in Bullseye, so this needs to be changed anyways.